src/common/interfaces.ts

```diff
@@ -52,35 +58,65 @@ export interface JobConfig {
   type: string;
 }
 
-export interface IngestionJobs {
+export interface IngestionJobsConfig {
   seed: JobConfig | undefined;
 }
 
-export interface PollingJobs {
+export interface IngestionPollingJobsConfig {
   [key: string]: JobConfig | undefined;
   new: JobConfig | undefined;
   update: JobConfig | undefined;
   swapUpdate: JobConfig | undefined;
+}
+
+export interface ExportPollingJobsConfig {
+  [key: string]: JobConfig | undefined;
   export: JobConfig | undefined;
 }
 
+export type PollingJobs = IngestionPollingJobsConfig | ExportPollingJobsConfig;
+
 export interface PollingTasks {
   init: string;
   finalize: string;
 }
 
 export interface PollingConfig {
   tasks: PollingTasks;
-  jobs: PollingJobs;
   maxTaskAttempts: number;
 }
 
+export interface TilesMergingTaskConfig {
+  type: string;
+  tileBatchSize: number;
+  taskBatchSize: number;
+  radiusBuffer: number;
+  radiusBufferUnits: Units;
+  truncatePrecision: number;
+  truncateCoordinates: number;
+}
+
 export interface TilesSeedingTaskConfig {
   type: string;
   grid: string;
   maxZoom: number;
   skipUncached: boolean;
 }
+
+export interface TilesExportingTaskConfig {
+  type: string;
+}
+
+export interface JobManagementConfig {
+  config: JobManagerConfig;
+  polling: PollingConfig;
+  ingestion: {
+    pollingJobs: IngestionPollingJobsConfig;
+    jobs: IngestionJobsConfig;
+    tasks: { tilesMerging: TilesMergingTaskConfig; tilesSeeding: TilesSeedingTaskConfig };
```

Comment:
maybe make a new type for ingestion task types

Reply:
done